src/mame/drivers/xavix.cpp

```diff
@@ -2010,6 +2036,8 @@ CONS( 2002, rad_bdp,   0,          0,  xavix,            rad_bdp,  xavix_state,
 
 CONS( 2002, rad_socr,  0,          0,  xavix,            epo_epp,  xavix_state,          init_xavix,    "Radica / Epoch / SSD Company LTD",             "Play TV Soccer", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_SOUND )
 
+CONS( 2002, epo_esdx,  0,          0,  xavix,            epo_epp,  xavix_state,          init_xavix,    "Epoch / SSD Company LTD",                      "Exciting Stadium DX (Japan)", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_SOUND )
```

Comment:
Per your screenshot on Twitter, this should be "Excite Stadium", not "Exciting Stadium".